### ISSUE TITLE
Are you afraid of Chinese government?

### ISSUE THREAD

**Issue by GAO23** [cirosantilli/chat#29]
Are you afraid that CCP will shut down your git account and kidnapped you to China to face treason trial?

**Comment by cirosantilli** [comment]
No. Are there any known cases of that happening outside of China.

**Comment by GAO23** [comment]
yeah, in HK, many writers had been kidnapped, some of them even hold foreign citizenship instead of Chinese nationality. I am more interested in knowing if the Chinese gov't will censor github like facebook because it has been used by Chinese netizens to bypass the censorship software, and in your case, voicing dissents on the internet.

**Comment by GAO23** [comment]
Oh, just like to mention that falun gong is an unpleasant cult. I can understand why you speak out against censorship but why do you sympathize with a group that... well, is pretty much this: http://rationalwiki.org/wiki/Falun_Gong

**Comment by cirosantilli** [comment]
Hi, extensive FLG discussion at: https://github.com/cirosantilli/china-dictatorship/blob/master/FAQ.md#falun-gong

HK: interesting, I didn't know.

GH block: I'm trying to block it and Stack Overflow. But I think it is unlikely to happen since there are so few users (only programmers).

**Comment by GAO23** [comment]
You support a religion that is homophobic and bigoted? Perhaps you dont know falun gong's history. it is founded in the 1990s by Li Hongzhi, a man who once claimed that aliens from outer space had invaded the hman society and wh preached pseudo science to his followers. Later, state media wrote a few articles ridiculing the practice and in response, mass of falun gong practitioners protested outside government compound. That's why they were banned.

**Comment by cirosantilli** [comment]
I don't support FLG, I support from of religion. I  know all of this. All religions are homophobic. Democracy is a religion. See FAQ.

**Comment by GAO23** [comment]
Democracy is not a religion. No one preaches it as if it's a dogma. You don't understand falun gong. Just as a fascism is a dangerous ideology that deserves persecution, falun gong is a dangerous cult that jeopardizes the Chinese society and hence deserves to be persecuted. I do find the extrajudicial arrest/torture/imprisonment against the falun gong by the Chinese authorities unethical and reprehensible but that is a different matter. I support its ban and frown on any one who defends such nonsensical cult.

**Comment by throwaway199** [comment]
@GAO23 
>deserves to be persecuted
You can say the same for every religion in China

**Comment by GAO23** [comment]
@throwaway199 
The government. They know what is the best approach to keep public order and preventing the country from descending into chaos.
